tests/runtest.sh

```diff
@@ -382,11 +383,17 @@ function create_core_overlay {
 }
 
 function precompile_overlay_assemblies {
+    declare -A skipCrossGenFiles
+
+    while read line
+    do
+      skipCrossGenFiles[$line]=1
+    done <  "$(dirname "$0")/skipCrossGenFiles.$ARCH.txt"
 
     if [ $doCrossgen == 1 ]; then
         local overlayDir=$CORE_ROOT
 
-        filesToPrecompile=$(ls -trh $overlayDir/*.dll)
+        filesToPrecompile=$(find -L $overlayDir -iname \*.dll -not -iname \*.ni.dll -not -iname \*-ms-win-\* -type f )
```

Comment:
Using find to reduce the number of crossgen calls.  

* Removed *.ni.dll from list.  
* Removed *-ms-win-* from the list as these represent the bulk of the native windows dll which are copied into the unix overlay.  It is possible this should just be a rm *-ms-win* when the overlay is created